src/tests/Microsoft.PowerFx.Connectors.Tests.Shared/PublicSurfaceTests.cs

```diff
@@ -59,9 +59,11 @@ public void PublicSurfaceTest_Connectors()
               "Microsoft.PowerFx.Connectors.OpenApiParser",
               "Microsoft.PowerFx.Connectors.PowerFxConnectorException",
               "Microsoft.PowerFx.Connectors.PowerPlatformConnectorClient",
+              "Microsoft.PowerFx.Connectors.RawTable",              
+              "Microsoft.PowerFx.Connectors.Relationship",
```

Comment:
Let's discuss - 
the goal is we have an abstract API and DV doesn't need to write against a connector specific API.

Reply:
Was only needed for tests in DV/NL repos
We can hide them for now